index.bs

```diff
@@ -461,6 +488,24 @@ bitmask value.
 
 <h2 id="event-design">Event Design</h2>
 
+<h3 id="dont-invent-event-like">Don't invent your own event listener-like infrastructure</h3>
+
+In some cases you might not have an object to inherit from `EventTarget` but
```

Comment:
It could be convenient to create a pair of APIs to "register"/"unregister", "addListener"/"removeListener", "subscribe"/"unsubscribe", start"/"stop", etc., that take a callback and get invoked multiple times until paired cancel API is called. Instead, use the existing event registration pattern and API controls to start/stop the underlying process (since event listeners should not have side-effects). If the callback would have been provided specific data, then this data should be added to an Event object (but see https://w3ctag.github.io/design-principles/#state-and-subclassing as this is not always necessary).

In some cases...

Reply:
In some cases, you can transfer the state that would be surfaced in callback parameters into a more persistent object which, in turn, can inherit from EventTarget.

Reply:
Comment for wrong PR :) but added to the right one